### DISCUSSION TITLE
Charges in comestibles and recipes

### DISCUSSION THREAD

**Discussion by actual-nh** [CleverRaven/Cataclysm-DDA#46489] [Q&A] [answered]
Exactly what does having "charges" (above 1) do in a recipe or item definition (not item spawn specification)? See #46469 for more background on why I'm asking.

**Comment by curstwist** [comment, marked as answer]
Should be the number of portions that are created or enclosed in the packaging.

**Reply by actual-nh** [reply]
OK; and is each _**portion**_'s nutrition given by the calories, etc - or is it the entire thing's nutrition? Thanks!
